src/test/java/no/difi/sdp/client2/internal/KvitteringBuilderTest.java

```diff
@@ -45,8 +45,8 @@ public void should_build_aapnings_kvittering() {
 
         AapningsKvittering aapningKvittering = (AapningsKvittering) kvitteringBuilder.buildForretningsKvittering(ebmsKvittering);
 
-        assertNotNull(aapningKvittering.getKonversasjonsId());
-        assertNotNull(aapningKvittering.getTidspunkt());
+        assertNotNull(aapningKvittering.kvitteringsinfo.getKonversasjonsId());
```

Comment:
Forsåvidt ikke relevant for dette PRet, men tar det med likevel: jeg er ikke så glad i sånne `notNull`-tester – bedre å teste den faktiske verdien den har fått